lib/tasks/TaskUtil.js

```diff
@@ -86,12 +86,16 @@ class TaskUtil {
 	 * This method is only available to custom task extensions defining
 	 * <b>Specification Version 2.2 and above</b>.
 	 *
-	 * @param {module:@ui5/fs.Resource} resource The resource the tag should be cleared for
+	 * @param {string|module:@ui5/fs.Resource} resourcePath Path or resource-instance the tag should be cleared for
 	 * @param {string} tag Tag
 	 * @public
 	 */
-	clearTag(resource, tag) {
-		return this._projectBuildContext.getResourceTagCollection().clearTag(resource, tag);
+	clearTag(resourcePath, tag) {
+		return this._projectBuildContext.getResourceTagCollection().clearTag(resourcePath, tag);
```

Comment:
```suggestion
		return this.getResourceTagCollection().clearTag(resourcePath, tag);
```

Reply:
Removed the (unused) getResourceTagCollection function